tcbsd-provision-playbook.yaml

```diff
@@ -44,8 +106,8 @@
       ansible.builtin.pip:
         name: "{{ tc_install_pip_packages }}"
 
-    - name: Install pip
-      # Packages only available via pip will be installed after this
+    - name: Unstall pip
```

Comment:
I swear I read the previous PR  😓

Reply:
I'm pretty upset at myself for `unstall` vs `uninstall` lol